osgi/impl/src/main/resources/karaf_dashboard_example.cdfde

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
Can the samples be put in a subfolder with a related name, instead of in the root of the bundle?